apps/web-giddh/src/app/services/contact.service.ts

```
@@ -192,8 +192,10 @@ export class ContactService {
     public getAccountStatementList(requestObj: any): Observable<BaseResponse<any, any>> {
         const model = requestObj.model ? requestObj.model : requestObj;
         const body = requestObj.body ? requestObj.body : null;
+        let requestObjCopy = { ...requestObj };
+        delete requestObj.branchUniqueName;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)